src/Microsoft.DotNet.Build.Tasks/Targets/coreAssembly.rsp

```diff
@@ -0,0 +1,2 @@
+﻿# CSC requires the runtimemetadataversion flag when building a core assembly (containing System.Object)
+/runtimemetadataversion:v4.0.30319
```

Comment:
Note this will break for anyone not using roslyn compiler. For example anyone building in VS2013. We should think about what we can do here.